src/OrchardCore/OrchardCore.Workflows.Abstractions/Models/WorkflowScriptSyntax.cs

```diff
@@ -0,0 +1,8 @@
+namespace OrchardCore.Workflows.Models
+{
+    public enum WorkflowScriptSyntax
+    {
+        JavaScript,
+        Liquid
```

Comment:
```suggestion
        Liquid,
```

Reply:
this will be the first enum with such convention, I've just checked and all enums in the project are without comma after last item, so I will leave it as is